Update enumerating-all-processes.md

aloofbit opened the pull request:
Hi, I was trying to use this code and most of my applications showed "<unknown>". I was wondering if we could add a note about x64/x64 mode applications. I'm really new to trying to use C++ so I am not sure if this is something that was supposed to be very obvious. Please let me know if there is a better way to do this!

aloofbit committed:
Update enumerating-all-processes.md

Hi, I was trying to use this code and most of my applications showed "<unknown>". I was wondering if we could add a note about x64/x64 mode applications. I'm really new to trying to use C++ so I am not sure if this is something that was supposed to be very obvious. Please let me know if there is a better way to do this!

PRMerger13 commented:
@aloofbit : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Karl-Bridge-Microsoft committed:
Update enumerating-all-processes.md

Thanks @aloofbit - I've revised your suggestion as it actually didn't qualify the function mentioned (EnumProcesses) in the first sentence. I've added a sentence that does mention EnumProcessModules and a note with the 64 bit qualification.

PRMerger12 commented:
@Karl-Bridge-Microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.
